spec/index.html

```diff
@@ -1791,8 +1789,12 @@ <h2>Changes between RDF 1.1 and RDF 1.2</h2>
   <ul>
     <li>Added <a href="#section-dataset-quad" class="sectionRef"></a>
       for informative definition of a <a>quad</a>.</li>
+    <!-- The following bullet point should be removed when Section 1.3 is changed
+         to introduce triple terms instead of quoted triples. -->
     <li>Added <a href="#section-quoted-triples" class="sectionRef"></a>
-      and definitions for <a>quoted triple</a> and <a>asserted triple</a>.</li>
+      <s>and definitions for <a>quoted triple</a> and <a>asserted triple</a></s>.</li>
```

Comment:
We may want something here, and in the Quoted Triples section, that this notion is deprecated and will be removed or changed when the WG resolves nomenclature.

Reply:
I created issue #81 as a reminder that Section 1.3 needs to be changed, and extended this PR (see commit https://github.com/w3c/rdf-concepts/pull/78/commits/195505a07a352e1d6fc4c2825dd5a408a50c7900) by adding two references to this issue, one below this bullet point here and the other one at the beginning of Section 1.3.